sql/item_mods.sql

```diff
@@ -26019,11 +26019,11 @@ INSERT INTO `item_mods` VALUES (23125,14,28);     -- CHR+28
 INSERT INTO `item_mods` VALUES (23125,23,23);     -- Attack+23
 INSERT INTO `item_mods` VALUES (23125,25,46);     -- Accuracy+46
 INSERT INTO `item_mods` VALUES (23125,29,6);      -- Magic Defense+6
-INSERT INTO `item_mods` VALUES (23125,31,72);     -- Magic Evasion+72
+INSERT INTO `item_mods` VALUES (23125,31,74);     -- ffxiah states Magic Evasion+74
 INSERT INTO `item_mods` VALUES (23125,68,69);     -- Evasion+69
 INSERT INTO `item_mods` VALUES (23125,384,40);    -- Haste +40
 INSERT INTO `item_mods` VALUES (23125,491,17);    -- Waltz Potency+17
-INSERT INTO `item_mods` VALUES (23125,497,1);     -- Waltz Recast+1
+INSERT INTO `item_mods` VALUES (23125,497,1);     -- Waltz Recast+1 (Should this be +1 or -1?)
```

Comment:
if it was already there the comment you did can stay (my eyes are not liking how githubs red looks like light pink on a  white background lately)